superset/__init__.py

```diff
@@ -160,11 +170,28 @@ def __call__(self, environ, start_response):
 for middleware in app.config.get('ADDITIONAL_MIDDLEWARE'):
     app.wsgi_app = middleware(app.wsgi_app)
 
+class PrefixMiddleware(object):
+
+    def __init__(self, app, prefix='/rasmi'):
+      self.app = app
+      self.prefix = prefix
+
+    def __call__(self, environ, start_response):
+
+       # if environ['PATH_INFO'].startswith(self.prefix):
+        environ['PATH_INFO'] = environ['PATH_INFO'][len(self.prefix):]
+
+        environ['SCRIPT_NAME'] = self.prefix
+        return self.app(environ, start_response)
+        # else:
+        #     start_response('404', [('Content-Type', 'text/plain')])
+        #     return ["This url does not belong to the app.".encode()]
 
+app.wsgi_app = PrefixMiddleware(app.wsgi_app)
 class MyIndexView(IndexView):
     @expose('/')
     def index(self):
-        return redirect('/superset/welcome')
+        return redirect(url_for('Superset.welcome'))
 
```

Comment:
is it convention of url_for to pass value as  Superset.welcome instead /superset/welcome
 or we can use same here